src/snapred/backend/dao/request/SimpleDiffCalRequest.py

```diff
@@ -8,4 +8,3 @@
 class SimpleDiffCalRequest(BaseModel):
     ingredients: DiffractionCalibrationIngredients
     groceries: Dict[str, str]
-    skipPixelCalibration: bool
```

Comment:
I think it'd actually be better to remove this from the ingredients instead.  It is only used within the service layer